### ISSUE TITLE
Use this for a blog post?

### ISSUE THREAD

**Issue by ElasticStewart** [josefjahn/tesla_dashboard_backend#1]
Hi @josefjahn ! I came across your repo here. I was planning to give it a spin with my Tesla, and perhaps write a blog about it all (I work for Elastic). 

I was curious if 1) you were OK with me doing that and sharing your repo here, and 2) if you have any updates/does it still work. Thank you!!

**Comment by dividebysandwich** [comment]
Hi there!

Sure, I'm more than happy if you do. I've updated the script to work with
the new API, though sadly it turns out Tesla has recently (just a few days
ago) changed their authentication yet again, this time adding a captcha
that prevents this from working completely headless. I haven't found a
solution that lets me run completely unattended yet.


You may also be interested in the various other dashboards I've made, for
example one that takes solar inverter and battery stats from my home energy
storage using a simple backend at
https://github.com/josefjahn/energydisplay-backend
....and then there's a bunch of ways to display the data, be that using
e-ink (https://github.com/josefjahn/energydisplay) or in a Kibana Dashboard:

![image](https://user-images.githubusercontent.com/23048489/129174723-b39d51b8-f8e2-4bbc-8a76-32e78faeee4d.png)

I'm also using Kibana for a bunch of other stuff around the house, for
example to monitor my Automower:
![image](https://user-images.githubusercontent.com/23048489/129174807-6478f9b0-e2ab-4bf9-919b-fb9a8ebd5d54.png)



I also have a 20GHz radar module pointed at the road and catch speeders :)

![image](https://user-images.githubusercontent.com/23048489/129174874-198eee6a-d895-49df-a804-ebb8f8a56ea9.png)



-Josef

**Comment by ElasticStewart** [comment]
Yep I experienced the same with the new API, what a bummer they did that!

Those are also so cool, thanks for sharing! I'll be in touch :)
